requirements.txt

```diff
@@ -1,3 +1,4 @@
 prefect>=2.7.2
+dbt_core>=1.1.1
```

Comment:
If we're making this a requirement that is always installed, we don't need the `installation_check` validator, right?